mkdocs.yml

```diff
@@ -37,9 +37,9 @@ markdown_extensions:
 
 nav:
   - 'Overview': index.md
-  - '0.X API':
-      - 'rxdogtag': 0.x/rxdogtag/com.uber.rxdogtag/
-      - 'rxdogtag-autodispose': 0.x/rxdogtag-autodispose/com.uber.rxdogtag.autodispose/
+  - '1.X API':
```

Comment:
Not sure if it's worth keeping 0.x API reference considering that nothing has changed. Thoughts?

Reply:
Yeah can remove after